src/ParallelAlgorithms/ApparentHorizonFinder/Storage.hpp

```diff
@@ -0,0 +1,166 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+#include <unordered_map>
```

Comment:
Check includes.  At minimum, need `<cstddef>`, `<set>`, and `<unordered_set>`.

Reply:
pushed fixups